crates/zksync/core/src/vm/db.rs

```diff
@@ -243,8 +243,13 @@ where
     }
 
     /// Load an account into the journaled state.
-    pub fn load_account(&mut self, address: Address) -> &mut Account {
-        self.ecx.journaled_state.load_account(address).expect("account could not be loaded").data
+    pub fn load_account(&mut self, address: Address) -> Account {
+        self.ecx
+            .journaled_state
+            .load_account(address)
+            .expect("account could not be loaded")
+            .data
+            .clone()
```

Comment:
Previously the `&mut Account` ref allowed the user to directly manipulate the account - this would break that functionality - how did foundry work around this change?